akka-docs/src/main/paradox/index.md

```diff
@@ -13,9 +13,11 @@
 * [stream/index](stream/index.md)
 * [index-network](index-network.md)
 * [discovery](discovery/index.md)
+* [discovery](discovery/index.md)
 * [coordination](coordination.md)
 * [index-utilities](index-utilities.md)
 * [common/other-modules](common/other-modules.md)
+* [deploying](deploying.md)
```

Comment:
Funny thing: while looking over the docs structure for 2.6 I made a note about perhaps moving things related to running Akka into one top-level section. I think this is a good idea!

Reply:
Great, I think it should be a top level topic, as easy to find in docs as getting started, for all the reasons.